src/Hamiltonians/Hamiltonians.jl

```diff
@@ -86,6 +88,8 @@ export hubbard_dispersion, continuum_dispersion
 export FroehlichPolaron
 export ParticleNumberOperator
 
+export MolecularHamiltonian, MolecularHamiltonianOperatorColumn
```

Comment:
```suggestion
export MolecularHamiltonian
```
I don't think the column struct needs to be exported. The user shouldn't have to deal with it.